docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Configure Teleport to Create Host Users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH Service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session. This feature has overlap with the [PAM](./ssh-pam.mdx) integration, however
+it provides some additional convenience of not having to manually configure a PAM
+script to create and delete users.
+
+It also has benefits over manually creating users as it allows for separate host
+users for each member of an organization. This allows for more fine grained control
```

Comment:
```suggestion
users for each member of an organization. This provides more fine-grained control
```
To avoid repeating "allows" and hyphenate "fine grained"